README.md

```diff
@@ -68,6 +68,18 @@ class Page extends React.Component {
 }
 ```
 
+You can also call the underlying `localize` function to pseudo-localize any string.
+
+
+```js
+import { localize } from 'pseudo-localization';
+// OR
+import localize from 'pseudo-localization/localize';
+
+console.log(localize('hello')); // --> ħḗḗŀŀǿǿ
+console.log(localize('hello', { strategy: 'bidi' })); // --> ɥǝʅʅo‬
```

Comment:
I think making the comment reflect how the string would display in a UI (rendered with the unicode-bidi algorithm) surrounded by right-to-left character marks would be good. 

```diff
- // --> ɥǝʅʅo 
+ // --> oʅʅǝɥ
```

That is consistent with the example in the [bidi](https://github.com/tryggvigy/pseudo-localization/blob/master/README.md#bidi---%C9%A5s%C4%B1%CA%85%C6%83u%C6%8E-%C4%B1p%C4%B1%D4%90) section header in the readme

Reply:
Oops, I just copied it from my terminal, didn't think to flip it. Done.